cloudify_ansible_sdk/__init__.py

```diff
@@ -76,190 +69,99 @@ def __init__(self,
-
         self.run_data = run_data or {}
-
-        self.inventory = self._set_inventory()
-        self.variable_manager = self._set_variable_manager()
-        self.options = self._set_options()
-        self.display.verbosity = self.options.verbosity
-
-        self.tqm = self._set_task_manager()
-        self.runner = self._set_runner()
+        self.environment_variables = environment_variables or {}
+        self._verbosity = verbosity
         self.logger = logger
 
-    def _set_inventory(self):
-        """Assign the inventory property."""
-        _kwargs = deepcopy(self.inventory_config)
-        if not isinstance(_kwargs, dict):
-            raise CloudifyAnsibleSDKError(
-                'inventory_config must be a dictionary.')
-        if 'loader' not in _kwargs:
-            _kwargs.update({'loader': self.loader})
-        if 'sources' not in _kwargs:
-            _kwargs.update({'sources': self.sources})
-        return InventoryManager(**_kwargs)
-
-    def _set_variable_manager(self):
-        """Assign the variable_manager property."""
-        _kwargs = deepcopy(self.variable_manager_config)
-        if not isinstance(_kwargs, dict):
-            raise CloudifyAnsibleSDKError(
-                'variable_manager_config must be a dictionary.')
-        if 'loader' not in _kwargs:
-            _kwargs.update({'loader': self.loader})
-        if 'inventory' not in _kwargs:
-            _kwargs.update({'inventory': self.inventory})
-        variable_manager = VariableManager(**_kwargs)
-        variable_manager.extra_vars = self.run_data
-        variable_manager.set_inventory(self.inventory)
-        return variable_manager
-
-    def _set_options(self):
-        """Assign the options variable."""
-        options_kwargs = deepcopy(self.options_config)
-        if not isinstance(options_kwargs, dict):
-            raise CloudifyAnsibleSDKError(
-                'options_config must be a dictionary.')
-        for k, v in self.options_defaults.items():
-            if not options_kwargs.get(k):
-                options_kwargs[k] = v
-        try:
-            return Options(**options_kwargs)
-        except TypeError as e:
-            raise CloudifyAnsibleSDKError(
-                'Invalid options_config: {0}'.format(str(e)))
-
-    def _set_task_manager(self):
-        return TaskQueueManager(
-            inventory=self.inventory,
-            variable_manager=self.variable_manager,
-            loader=self.loader,
-            options=self.options,
-            passwords=self.passwords)
-
-    def _set_runner(self):
-        runner = PlaybookExecutor(
-            playbooks=[self.site_yaml_path],
-            inventory=self.inventory,
-            variable_manager=self.variable_manager,
-            loader=self.loader,
-            options=self.options,
-            passwords=self.passwords)
-        setattr(runner, '_tqm', self.tqm)
-        return runner
+        self.logger.error(
```

Comment:
@EarthmanT Why you are adding `error` logger inside the `__init__` method ??

Reply:
why not? @mabuaisha

Reply:
I did not focus on the message, I though you are forget to remove or so, I will approve now